api/envoy/extensions/filters/common/rbac/matcher/upstream/v3/upstream_ip_matcher.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.common.rbac.matcher.upstream.v3;
```

Comment:
This is the wrong location; this should be in something like `envoy.extensions.rbac.matcher.upstream.v3`. All paths for protos and files will need adjusting.